packages/combo-box/test/toggling-dropdown.test.js

```diff
@@ -193,20 +201,13 @@ describe('toggling dropdown', () => {
   });
 
   describe('closing', () => {
-    (isIOS ? it : it.skip)('should close popup when clicking outside overlay', () => {
-      comboBox.open();
-
-      click(document.body);
-
-      expect(comboBox.opened).to.be.false;
-    });
-
-    (isIOS ? it.skip : it)('should close popup when clicking outside overlay', () => {
```

Comment:
These two tests are duplicates, removed one and updated another one to use `outsideClick()` helper.